CHANGELOG.md

```diff
@@ -1,5 +1,16 @@
 # Qiita changelog
 
+Version 092020
+--------------
+
+* Added a new endpoint to inject artifacts to existing preparations or jobs: `/qiita_db/artifact/`
+* Now outdated commands with the exact same name than newer commands will be marked as not outdated. This is helpful for cases where the commands haven't changed between version
+* Added the `add_ebi_accessions` to the to_dataframe() method of the information files so it can be used to `redbiom`. This will allow searching via sample or experiment accessions
```

Comment:
```suggestion
* Added the `add_ebi_accessions` to the `to_dataframe()` method of the information files so it can be used to `redbiom`. This will allow searching via sample or experiment accessions
```